tests/cupy_tests/core_tests/test_raw.py

```diff
@@ -1190,23 +1198,17 @@ def test_raw_picklable(self):
         elif self.raw == 'mod_ker':
             # pickle the RawKernel fetched from the RawModule
             obj = self.mod.get_function('test_sum')
-        with open(self.temp_dir + '/TestRawPicklable', 'wb') as f:
+        with open(self.temp_dir + '/raw.pkl', 'wb') as f:
```

Comment:
I thought one of the issues was that we need to handle the paths in a more robust way, using `os.path.join`?

Reply:
Ditto for below.

Reply:
I agree that using `os.path.join` is the cleaner way but that was not a problem this time.
The reason why it's failing only in Windows is that `string` template is used to embed a file path. Windows file paths contain backslashes, so when embedding a path (or any arbitrary strings) into a string literal in Python code template, we need to escape `\` as `\\`. Obviously, we don't want to do this 😅

Reply:
Rename (`TestRawPicklable` -> `raw.pkl`) itself has no meaning (sorry for confusion), I just wanted to make sure that it's a pickle file.

Reply:
I see, thanks for explanation! 👍